readme.md

```diff
@@ -0,0 +1 @@
+# fix linters errors
```

Comment:
- [ ] The README file should be descriptive, kindly consider using the Microverse README template so your work can be professionally documented